swap_meet/clothing.py

```diff
@@ -0,0 +1,15 @@
+from swap_meet.item import Item
+
+class Clothing(Item): 
+    category = "Clothing"
+    
+    def __init__(self,condition=0,age=0): 
```

Comment:
Style: put a space after commas in parameter lists:

```suggestion
    def __init__(self, condition=0, age=0): 
```